test/libsolidity/syntaxTests/inheritance/override/interface_exception/diamond_needed.sol

```diff
@@ -0,0 +1,10 @@
+interface I {
+	function f() external;
+}
+interface J {
+	function f() external;
+}
+contract C is I, J {
+	function f() external override (I, J) {}
```

Comment:
The test is only meaningful if you do not supply `override (I, J)` here, isn't it?

Reply:
This PR didn't add the support for leaving out override with multiple parameters. Should it be added?

Reply:
No, I'm saying that you are not testing the property with the test.